packages/waf.rb

```diff
@@ -3,25 +3,12 @@
 class Waf < Package
   description 'The Waf build system'
   homepage 'https://waf.io/'
-  version '1.9.13'
+  version '1.9.13-1'
```

Comment:
New version?  See https://gitlab.com/ita1024/waf/-/archive/waf-2.0.22/waf-waf-2.0.22.tar.bz2

Reply:
What is wrong with the new version?

Reply:
Actually, the version doesn't match the source.  Can we update?